docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -120,13 +120,18 @@ A specific version can be used by [specifying a custom kubectl location](/docs/d
 
 ## Installing Software On Dynamic Workers
 
-It is possible to install additional software on dynamic workers.  By default, every worker is destroyed after it has been allocated for over 72 hours.  Because of that, if you choose to install additional software, you are responsible for:
+Octopus does not recommend installing additional software on Dynamic Workers. 
+
+By default, every dynamic worker is destroyed after it has been allocated for over 72 hours. In addition Octopus cannot guarantee that the dynamic worker leased to run one step will be the same worker leased to other executing steps in a deployment or runbook run. 
+
+For deployments and runbook runs that require additional software dependencies on a Dynamic worker, our recommendation is to leverage [execution containers for workers](docs/projects/steps/execution-containers-for-workers/index.md).  Octopus provides execution containers with a baseline of tools pre-installed.  However, we can't create a container with every possible software combination you might need.  Please see this [blog post](https://octopus.com/blog/tips-building-custom-execution-containers) for more information on how to create custom execution containers.
```

Comment:
Might be worth calling out that dynamic workers have the latest worker-tools image cached when they're instantiated? Makes it an even easier win to use execution containers over custom tooling when possible.